src/jdk.jdwp.agent/share/native/libjdwp/debugInit.c

```diff
@@ -74,6 +79,9 @@ static jboolean initOnUncaught = JNI_FALSE; /* init when uncaught exc thrown */
 
 static char *launchOnInit = NULL;           /* launch this app during init */
 static jboolean suspendOnInit = JNI_TRUE;   /* suspend all app threads after init */
+#if defined(J9VM_OPT_CRIU_SUPPORT)
```

Comment:
This requires `#include "j9cfg.h"` which means a dependency on https://github.com/ibmruntimes/openj9-openjdk-jdk/pull/596.

I see that include in `debugInit.h`: why is `debugInit.h` not included here?

Reply:
This PR was based on an earlier PR
* https://github.com/ibmruntimes/openj9-openjdk-jdk/pull/592
https://github.com/ibmruntimes/openj9-openjdk-jdk/pull/596 is an improvement though backporting of this PR depends on both PRs.

Note: this file includes `util.h` which includes `debugInit.h` indeed.

Reply:
I think every file that uses any flag from `j9cfg.h` should include that header file directly, otherwise we will silently omit code guarded with `#if defined(J9VM_OPT_CRIU_SUPPORT)`, for example, leading to inconsistencies.

Reply:
Do we want to include `j9cfg.h` in both `debugInit.c` and `debugInit.h`, similar comments for other c source/header files?